templates/posttypes/contributor.blade.php

```diff
@@ -3,82 +3,84 @@
 		@if ( $contributor['contributor_picture'] )
 			<img class="contributor__profile__picture" alt="Contributor photo" src="{{ $contributor['contributor_picture'] }}"/>
 		@endif
-		<p class="contributor__name">
-			@if ( !isset( $exporting ) )
-				<span class="screen-reader-text">name: </span>
-			@endif
-			{{ $contributor['name'] }}
-		</p>
-		@if ( $contributor['contributor_institution'] )
-			<p class="contributor__institution">
-				@if ( !isset( $exporting ) )
-					<span class="screen-reader-text">institution: </span>
-				@endif
-				{{ $contributor['contributor_institution'] }}
-			</p>
-		@endif
-		@if ( $contributor['contributor_user_url'] )
-			<p class="contributor__website">
+		<div class="{{ $contributor['contributor_picture']  ? 'contributor__info__small' : 'contributor__info__large' }}">
```

Comment:
Wrapping everything but the profile picture, so we can have a better control of the info block.